lib/Controller/SettingsController.php

```diff
@@ -129,6 +125,8 @@ public function addClient() {
 
 		$allowSubdomains = $this->request->getParam('allow_subdomains', null) !== null;
 		$client->setAllowSubdomains($allowSubdomains);
+		$trusted = $this->request->getParam('trusted', null) !== null;
```

Comment:
This will always set the client as trusted if the web sends the "trusted" param even if it sends it with a false value. If that's what we want, please include a comment in the code to clarify it, because it might be suspicious if a send "trusted=false" and the client is trusted.

Reply:
if so - same applies to allowSubdomains as well ..... will test ...

Reply:
Maybe, from the web UI, if the checkbox is checked is sends "trusted=1" (or something like that) and if it isn't checked it doesn't send anything. That could explain the behaviour from the web UI.
Anyway, this needs to be clarified because as things are coded right now, "trusted=false" would set the client as trusted.

In addition, this should probably match with whatever is implemented in the cli to avoid confusion.